android/sdk/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <bool name="isTablet">false</bool>
```

Comment:
Why is it always false in sdk?

Reply:
it's not always false
check android/sdk/src/main/res/values-sw600dp/attrs.xml

same loggic
just moved files from app to sdk